src/machines/java/com/enderio/machines/common/item/BoundSoulBlockItem.java

```diff
@@ -24,7 +27,7 @@ public BoundSoulBlockItem(Block pBlock, Properties pProperties) {
     public void addCommonTooltips(ItemStack itemStack, @Nullable Player player, List<Component> tooltips) {
         // TODO: NEO-PORT: Instead of having an item class, have a tooltip event for anything with stored entity data?
```

Comment:
We should probably move to this already, especially because of the addition of tag